packages/datagrid/src/components/__snapshots__/datagrid.test.js.snap

```diff
@@ -0,0 +1,268 @@
+            "comments": Array [],
+            "quality": -1,
+            "value": "33929307",
+          },
+          "sample.field6": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 1,
+            "value": "10920487",
+          },
+          "sample.field7": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 1,
+            "value": "11030795",
+          },
+          "sample.field8": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 0,
+            "value": "10771660",
+          },
+          "sample.field9": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 1,
+            "value": "10880464",
+          },
+        },
+        Object {
+          "index.index": 1,
+          "sample.field0": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 1,
+            "value": "Aguilcourt-Variscourt",
+          },
+          "sample.field1": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 0,
+            "value": "171702",
+          },
+          "sample.field2": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 1,
+            "value": "2190",
+          },
+          "sample.field3": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": -1,
+            "value": "c",
+          },
+          "sample.field4": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 1,
+            "value": "9560",
+          },
+          "sample.field5": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 0,
+            "value": "",
+          },
+          "sample.field6": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 1,
+            "value": "9812",
+          },
+          "sample.field7": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 0,
+            "value": "",
+          },
+          "sample.field8": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": 1,
+            "value": "8919",
+          },
+          "sample.field9": Object {
+            "avro": Object {},
+            "comments": Array [],
+            "quality": -1,
+            "value": "",
+          },
+        },
+      ]
+    }
+    rowHeight={39}
+    rowSelection="single"
+    suppressDragLeaveHidesColumns={true}
+    tabToNextCell={[Function]}
+  />
+</div>
+`;
+
+exports[`#injectedCellRenderer should injected the cell renderer 1`] = `
+<Component
+  avroRenderer={Object {}}
+  getComponent={[Function]}
+  id="injectedComponent"
+/>
```

Comment:
Please look at the snapshots when you test. This doesn't check what you're supposed to check
(This applies to 4 tests here)